src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -23,6 +23,7 @@
 using Microsoft.Extensions.Options;
 using Xunit;
 using Xunit.Sdk;
+using HttpProtocols = Microsoft.AspNetCore.Server.Kestrel.Core.HttpProtocols;
```

Comment:
Some type name conflicts.

HttpProtocols vs HttpVersions?

Reply:
That would conflict with System.Net.HttpVersions. How about HttpProtocol? Subtle, I know.

Reply:
System.Net has HttpVersion (no s)

https://docs.microsoft.com/en-us/dotnet/api/system.net.httpversion?view=netcore-3.1

Reply:
HttpProtocol?